# DP: test_sundry: Don't fail on import of the profile and pstats module

--- a/Lib/test/test_sundry.py
+++ b/Lib/test/test_sundry.py
@@ -57,7 +57,11 @@
             import opcode
             import os2emxpath
             import pdb
-            import pstats
+            try:
+                import pstats	# separated out into the python-profiler package
+            except ImportError:
+                if test_support.verbose:
+                    print("skipping profile and pstats")
             import py_compile
             import sched
             import sndhdr
